### PR TITLE
Readd win

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,22 +21,23 @@ requirements:
...
-    - giftext -h  # [not win]
-    - gifclrmp -h  # [not win]
-    - gif2rgb.exe -h  # [win]
-    - gifbuild.exe -h  # [win]
-    - gifclrmp.exe -h  # [win]
-    - gifecho.exe -h  # [win]
-    - giffix.exe -h  # [win]
-    - gifinto.exe -h  # [win]
-    - giftext.exe -h  # [win]
+    # - gif2rgb -h  # [not win]
```
Comment (isuruf) [inline recipe/meta.yaml]: Why remove these?

Comment (wolfv) [reply recipe/meta.yaml]: I don’t think they’re used and or packaged on Linux distros commonly.

I have them working on windows but it requires to add a copy of getopt.c for one of the exes.

Do you prefer that?

Comment (isuruf) [reply recipe/meta.yaml]: Yes, that's fine provided you include the License.

Comment (isuruf) [reply recipe/meta.yaml]: Linux distros package them separately. For eg: giflib-tools on Debian

**Comment by wolfv** [comment]
thanks @isuruf!

File: recipe/CMakeLists.txt [hunk 2]
```diff
@@ -70,10 +72,18 @@ file(GLOB giflib_MAN doc/*.1)
...
 add_library(gif SHARED ${giflib_SRC})
-target_link_libraries(gif m)
+
+if (NOT WIN32)
+    target_link_libraries(gif m)
+endif()
+
 set_target_properties(gif PROPERTIES VERSION ${LIBVER} SOVERSION ${LIBMAJOR})
 if(WIN32)
-    set_target_properties(gif PROPERTIES RUNTIME_OUTPUT_NAME "gif-${LIBMAJOR}")
```
Comment (isuruf) [inline recipe/CMakeLists.txt:76]: Why remove this?

Comment (wolfv) [reply recipe/CMakeLists.txt:76]: had problems linking, not sure if it was related to this in the end. let's see...

File: recipe/meta.yaml [hunk 3]
```diff
@@ -32,9 +38,7 @@ test:
     - gif2rgb.exe -h  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you remove these lines specific for win and make `gif2rgb -h` be used in unix and win?

**Comment by wolfv** [comment]
Ok that worked! 

@isuruf do you have input regarding the license? The getopt I am currently using is LGPL licensed (https://github.com/Chunde/getopt-for-windows/blob/master/getopt.c)

There is also a BSD licensed version: https://github.com/kimgr/getopt_port

I don't know where and how to add the license if required?

File: recipe/meta.yaml (hunk 3 above)
Comment (wolfv) [reply recipe/meta.yaml]: done

**Review by 183amir — APPROVED** [review]
Hi @wolfv feel free to add yourself as the maintainer as well.

**Comment by isuruf** [comment]
@wolfv, can you use a MIT or BSD licensed one? Something like, https://github.com/conda-forge/gsl-feedstock/blob/master/recipe/getopt.h
As for the license, put it in the recipe folder.

**Comment by wolfv** [comment]
I guess this is ready to be merged now :)

**Pull request merged**